projects/packages/forms/src/contact-form/class-util.php

```diff
@@ -287,6 +279,24 @@ public static function jetpack_tracks_record_grunion_pre_message_sent( $post_id,
 			'feedback_id'     => esc_attr( $all_values['feedback_id'] ),
 		);
 
+		// old deprecated attribute that should been removed in May 2020.
+		if (
+			! isset( $extra_values['is_block'] )
+			|| ! $extra_values['is_block']
+		) {
+			$event_props['is_block'] = 0;
+		} else {
+			$event_props['is_block'] = 1;
```

Comment:
Should we set it to a boolean instead? `is_block` seems to be using booleans in other places.